test/hoverable-init.test.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env mocha, sinon, proclaim */
+import proclaim from 'proclaim';
+import sinon from 'sinon/pkg/sinon';
+import * as fixtures from './helpers/fixtures';
+
+const Hoverable = require('./../main.js');
+
+describe("o-hoverable init", () => {
+
+	afterEach(function() {
```

Comment:
I've had to add this to tidy up the `o-hoverable` modifications to the DOM after each of the auto-init tests have ran, I think this is the best solution but happy to take suggestions